test/jdk/java/awt/TrayIcon/ActionEventTest/ActionEventTest.java

```diff
@@ -48,27 +44,25 @@
 import java.awt.image.BufferedImage;
 
 public class ActionEventTest {
-
-    Image image;
     TrayIcon icon;
     Robot robot;
     boolean actionPerformed;
 
     public static void main(String[] args) throws Exception {
         if (!SystemTray.isSupported()) {
             System.out.println("SystemTray not supported on the platform." +
-                " Marking the test passed.");
+                    " Marking the test passed.");
         } else {
             if (System.getProperty("os.name").toLowerCase().startsWith("win")) {
                 System.err.println(
-                    "Test can fail on Windows platform\n"+
-                    "On Windows 7, by default icon hides behind icon pool\n" +
-                    "Due to which test might fail\n" +
-                    "Set \"Right mouse click\" -> " +
-                    "\"Customize notification icons\" -> \"Always show " +
-                    "all icons and notifications on the taskbar\" true " +
-                    "to avoid this problem.\nOR change behavior only for " +
-                    "Java SE tray icon and rerun test.");
+                        "Test can fail on Windows platform\n" +
+                                "On Windows 7, by default icon hides behind icon pool\n" +
+                                "Due to which test might fail\n" +
+                                "Set \"Right mouse click\" -> " +
+                                "\"Customize notification icons\" -> \"Always show " +
+                                "all icons and notifications on the taskbar\" true " +
+                                "to avoid this problem.\nOR change behavior only for " +
+                                "Java SE tray icon and rerun test.");
```

Comment:
~~I'd rather keep them aligned as they are — easier to read.~~

Shall we remove this at all?

The test is automatic, isn't it? No one will see these messages unless the test fails. [The instructions](https://wiki.openjdk.org/display/ClientLibs/Automated+client+GUI+testing+system+set+up+requirements#AutomatedclientGUItestingsystemsetuprequirements-Windows-SpecificSystemsetupnotes) for setting up Windows hosts to run client tests include:

> Show all icons and notifications in taskbar and don't hide them  
> Right Click on Taskbar → Properties → (Notification area)  
> Click Customize → Enable checkbox "Always show all icons and notifications on the taskbar"